Documentation/5.4.1/README.rst

```diff
@@ -1568,6 +1645,43 @@ ironic-inspector.yaml for Ironic-Inspector Deployment
       IronicInspectorExtraProcessingHooks: extra_hardware,lldp_basic,local_link_connection,nuage_lldp
 
 
+ovs-hw-offload.yaml
+~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    # A Heat environment file that enables OVS Hardware Offload in the overcloud.
+    # This works by configuring SR-IOV NIC with switchdev and OVS Hardware Offload on
+    # compute nodes. The feature supported in OVS 2.8.0
+
+    resource_registry:
+      OS::TripleO::Services::NeutronSriovHostConfig: ../puppet/services/neutron-sriov-host-config.yaml
+
+    parameter_defaults:
+
+      NovaSchedulerDefaultFilters: ['RetryFilter','AvailabilityZoneFilter','RamFilter','ComputeFilter','ComputeCapabilitiesFilter','ImagePropertiesFilter','ServerGroupAntiAffinityFilter','ServerGroupAffinityFilter','PciPassthroughFilter']
+      NovaSchedulerAvailableFilters: ["nova.scheduler.filters.all_filters","nova.scheduler.filters.pci_passthrough_filter.PciPassthroughFilter"]
+
+      # Kernel arguments for ComputeSriov node
+      ComputeSriovParameters:
+        KernelArgs: "intel_iommu=on iommu=pt pci=realloc"
+        #NOTE: By default TunedProfileName is set to "cpu-partitioning" in sriov-role.yaml.
+        # If IsolCpusList is not set in your environment, then leave TunedProfileName below to set to empty string.
+        # If planning on setting IsolCpusList in your environment
+        #   1. You can comment the below line to set TunedProfileName to "cpu-partitioning" or
+        #   2. You can pass your custom Tuned Profile to apply to the host
+        TunedProfileName: ""
+        OvsHwOffload: True
+        # Number of VFs that needs to be configured for a physical interface
+        NeutronSriovNumVFs: ["enp23s0f1:4:switchdev"]
+        # Mapping of SR-IOV PF interface to neutron physical_network.
+        # In case of Vxlan/GRE physical_network should be null.
+        # In case of flat/vlan the physical_network should as configured in neutron.
+        NovaPCIPassthrough:
+          - devname: "enp23s0f1"
```

Comment:
How are multiple ports specified; example dual port NIC cards.

Reply:
@shivsg, currently we can only use 1 smart nic for VRS offload. 

cc @sunny-verma @vmathur

Reply:
Do we support 1 NIC or 1 port. The NIC we have has 2 ports.